.readthedocs.yaml

```diff
@@ -1,11 +1,18 @@
+# Configuration on how ReadTheDocs (RTD) builds our documentation
+# ref: https://readthedocs.org/projects/2i2c-pilot-hubs/
+# ref: https://docs.readthedocs.io/en/stable/config-file/v2.html
+#
 version: 2
 
-conda:
-  environment: docs/environment.yml
-
 build:
-  image: latest
+  os: ubuntu-20.04
+  tools:
+    # NOTE: This must be mambaforge or it will mistakenly try to invoke `python` to do a conda install
+    # ref: https://stackoverflow.com/questions/69774313/why-does-rtd-run-python-env-create-instead-of-conda-env-create-when-building
+    python: "mambaforge-4.10"
```

Comment:
We may want to still use miniconda here instead of mambaforge... I am not against mamba, it is great! But we do not use it anywhere else in the codebase, AFAIK. But we do use conda.
I may be splitting hairs here but I aim for env management consistency, ja ja 😜

Reply:
sounds good, changed to conda